packages/react-next/RELEASE_NOTES.md

```diff
@@ -150,10 +150,10 @@ TODO: Diff of OUFR vs date-time DatePicker
 ### Pivot
 
 - Removed deprecated and redundant props from v7, including: `initialSelectedKey` and `defaultSelectedIndex`. Use `selectedKey` or `defaultSelectedKey` to define the selected tab, and provide `itemKey` on pivot item children.
-  - Removed deprecated styles prop `linkIsSelected?: boolean;`.
-  - Removed styles prop `rootIsLarge` and added `linkSize` instead.
-  - Removed styles prop `rootIsTabs` and added `linkFormat` instead.
-  - TODO: enumerate all removed props
+- Style props changes
```

Comment:
(nit) I got confused that this was talking about style-related props on IPivotProps, instead of IPivotStyleProps. Perhaps clarify?
```suggestion
- `IPivotStyleProps` changes
```